capa/main.py

```diff
@@ -933,6 +935,11 @@ def main(argv=None):
             logger.error("Input file '%s' is not a valid PE file: %s", args.sample, str(e))
             return E_CORRUPT_FILE
 
+        is_dotnet = is_dotnet_file(file_extractor.pe)
```

Comment:
main is quite long an complex already, dotnet would add more branches in various places

Reply:
hmm good question. dnfile invokes pefile so currently we are invoking pefile's parsing twice (when processing .NET PEs). I'm unsure if this is a performance hit great enough to warrant complicating main with more logic. I'd say for now we don't invoke dnfile until we know we are processing a .NET PE. So IMO using pefile here is fine.